lib/agenda/stop.js

```diff
@@ -18,8 +18,21 @@ module.exports = function(cb) {
   const _unlockJobs = function(done) {
     debug('Agenda._unlockJobs()');
     const jobIds = self._lockedJobs.map(job => job.attrs._id);
-    debug('about to unlock jobs with ids: %O', jobIds);
-    self._collection.updateMany({_id: {$in: jobIds}}, {$set: {lockedAt: null}}, done);
+
+    if (jobIds.length >= 1) {
```

Comment:
BTW for readability might be worth considering exiting early if you don't mind multiple return points:
```js
if (!jobIds.length) {
  debug('no jobs to unlock');
  return done();
}

// ...
return done();
```

But I'm quite indifferent regarding these.